doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -333,7 +333,7 @@ Keys samples
 Matter samples
 --------------
 
-|no_changes_yet_note|
+* Changed Bluetooth Low Energy variant of the Soft Device Controller (SDC) to use the Peripheral-only role in all Matter samples.
```

Comment:
```suggestion
* Changed the Bluetooth Low Energy variant of the Soft Device Controller (SDC) to use the Peripheral-only role in all Matter samples.
```

Reply:
I will add it in the following commit to not waste the CI time, ok?